### ISSUE TITLE
Help with owned books

### ISSUE THREAD

**Issue by PRElias** [adamkrogh/goodreads-dotnet#12]
Hello

This is actually a question: Is there any implementation of "owned books" method?

**Comment by VladimirRybalko** [comment]
Hi @PRElias 
Thanks for your question.
Project is an active development phase now. We just migrated to .NET Standard 2.0 and currently working with implementations of all Goodreads endpoints. 

"Owned books" method is there.
Client supports below methods:

- owned_books.create   —   Add to books owned.
- owned_books.list   —   List books owned by a user.
- owned_books.show   —   Show an owned book.
- owned_books.destroy   —   Delete an owned book.

Also Goodreads API has a bug in owned_books.update endpoint. Look at #9 for more details.

However, implementation of OwnedBook endpoint does not include into the current nuget package yet. We are going to release a new version soon.
So now you could clone a repository and build it yourself. (You should use VS2017 + .NET Core 2.0 SDK).
But keep in mind the master branch would be unstable now. We didn't test all methods yet.

Some examples of OwnedBooksEndpoint:
``` csharp
var authClient = GoodreadsClient.CreateAuth(apiKey, apiSecret, accessToken.Token, accessToken.Secret);
const long userId  = <Desired user id>;
var books = await authClient.GetOwnedBooks(userId);

long ownedBookInfoId = <Desired ownedBookId>
var ownedBook= await authClient.GetOwnedBookInfo(ownedBookInfoId);
```

Generally, you could find all available "owned_books" method [here](https://github.com/adamkrogh/goodreads-dotnet/blob/master/Goodreads/Endpoints/Implementations/OwnedBooksEndpoint.cs)

**Comment by PRElias** [comment]
Thank you for the answer and support.

**Comment by VladimirRybalko** [comment]
Hello @PRElias 
I just completed last changes in our library.
I think that Goodreads dotnet is ready for release.
I hope it will be very soon. We are going to create a new nuget package. 
Unfortunately, many broken changes were included into release. So the new nuget package is not compatible with old APIs. 
On the other hand, I did a lot of jobs to improve our API and make it more user friendly.
I hope you will like our new application.
